index.js

```diff
@@ -1,13 +1,138 @@
-const express = require('express')
-const { getSightings } = require('./utils.js')
-require('dotenv').config()
-
-const PORT = process.env.PORT;
+import express from "express";
+import { getSightings } from "./utils.js";
+import dotenv from "dotenv";
+import User, { sequelize } from "./models/User.js";
+import Post from "./models/Post.js";
+dotenv.config();
 const app = express();
+app.use(express.json());
+(async () => {
+  try {
+    await sequelize.authenticate();
+    console.log("Database connection has been established successfully.");
+  } catch (error) {
+    console.error("Unable to connect to the database:", error);
+  }
+})();
+const PORT = process.env.PORT || 5000;
+app.get("/users", async (req, res) => {
+  try {
+    const users = await User.findAll();
+    res.status(200).json(users);
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("Internal Server Error");
+  }
+});
+
+app.get("/users/:id", async (req, res) => {
+  try {
+    const user = await User.findByPk(req.params.id);
 
+    if (user) {
+      res.status(200).json(user);
+    } else {
+      res.status(404).send("User not found");
+    }
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("Internal Server Error");
+  }
+});
 app.get("/sightings", async (req, res) => {
   const sightings = await getSightings();
-  res.json(sightings);
+  res.status(200).json(sightings);
+});
+
+app.post("/sightings", async (req, res) => {
+  const newSighting = req.body; // Assuming the request body contains the new sighting
```

Comment:
We should never assume on the BE :)! Always make sure by checking the necessary properties first. If they are not provided, you return an error. If they are provided, go ahead with your happy path :)